integration/test/Test/LegalHold.hs

```diff
@@ -992,3 +992,60 @@ testBlockCreateMLSConvForLHUsers = do
       >>= \mp ->
         postMLSCommitBundle mp.sender (mkBundle mp)
           `bindResponse` assertLabel 409 "mls-legal-hold-not-allowed"
+
+testLHApiV1 :: App ()
+testLHApiV1 = do
```

Comment:
(as discussed on the phone: feel free to ignore)

Reply:
or just put an `assert (maxBound == V1) (pure ())` to the beginning of the test with a comment. then when version 2 comes, this will fail and we won't forget to revisit this test and make it's fine to not run it on v2.